rand_float.slang

```diff
@@ -0,0 +1,58 @@
+uniform float seed;
+RWStructuredBuffer<float> outputBuffer;
+
+uint seedPerThread(uint idx)
+{
+    return ((uint)idx + (uint)(seed * 1000000)) * 1099087573UL;
+}
+
+uint tauStep(uint z, uint s1, uint s2, uint s3, uint M)
+{
+    uint b = (((z << s1) ^ z) >> s2);
+    return (((z & M) << s3) ^ b);
+}
+
+[shader("compute")]
+[numthreads(64, 1, 1)]
+void randFloatMain(uint2 dispatchThreadId : SV_DispatchThreadID)
```

Comment:
can we rename it to `computeMain`?
It will be the 3rd category of entry point function.

Reply:
Renamed